src/index.html

```diff
@@ -11,7 +11,115 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset>
+        <legend>Personal information
+        </legend>
```

Comment:
It'd be better to put text on a new line
```suggestion
        <legend>
          Personal information
        </legend>
```
or you can put it in the same line:
```suggestion
        <legend>Personal information</legend>
```
 fix it for all legends and labels